stac_mjx/io.py

```
@@ -12,7 +12,18 @@
 import h5py
 from pathlib import Path
 from omegaconf import DictConfig
+from jax.lib import xla_bridge
 import os
```

Ruff

Redefinition of unused `os` from line 3

Remove definition: `os`

(F811)